dev-util/xxdiff/metadata.xml

```diff
@@ -2,7 +2,8 @@
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
 	<!-- maintainer-needed -->
+	<use><flag name="scripts">Python scripts that is provided around xxdiff</flag></use>
```

Comment:
What are they for, though?

Reply:
sry for delay. thanks for merging.

Reply:
No need to apologise, no one should be in front of their screens for the whole day. ;)